ui/app/rif/pages/domains/register/index.js

```diff
@@ -0,0 +1,375 @@
+      const costInRif = costInWei / 1e18;
+      this.props.showThis({
+        ...this.props,
+        currentStep: 'register',
+        yearsToRegister: yearsToRegister,
+        costInRif: costInRif,
+        gasCost: 0,
+      });
+    }
+  }
+
+  getBody (currentStep) {
+    const waitingForConfirmation = this.props.waitingForConfirmation;
+    const partials = {
+      available: (
+        <div className="domainRegisterAnimation">
+          <svg
+            className="checkmark"
+            xmlns="http://www.w3.org/2000/svg"
+            width="96"
+            height="96"
+            viewBox="0 0 52 52">
+            <circle className="checkmark__circle" cx="26" cy="26" r="25" fill="none" />
+            <path className="checkmark__check" fill="none" d="M14.1 27.2l7.1 7.2 16.7-16.8" />
+          </svg>
+          <div>
+            Domain Available
+          </div>
+        </div>
+      ),
+      register: (
+        <div className="domainRegisterInitiate">
+          <div>
+            You are going to renew {this.props.domainName}
+          </div>
+          <div>
+            <span>Number of years:</span>
+            <span className="hand-over" onClick={() => this.changeYearsToRegister(this.props.yearsToRegister - 1)}>
+              <i className="fa fa-minus"/>
+            </span>
+            <span>{this.props.yearsToRegister}</span>
+            <span className="hand-over" onClick={() => this.changeYearsToRegister(this.props.yearsToRegister + 1)}>
+              <i className="fa fa-plus"/>
+            </span>
+          </div>
+          <div>
+            <span>Cost in RIF:</span><span>{this.props.costInRif}</span><span>RIF</span>
+          </div>
+          <div>
+            <span>Gas Cost:</span><span>{this.props.gasCost}</span><span>RBTC</span>
+          </div>
+        </div>
+      ),
+      registering: (
+        <div className="domainRegisterProgress">
+          <div>
+            <article className="clock">
+              <div className="minutes-container">
+                <div className="minutes"/>
+              </div>
+              <div className="seconds-container">
+                <div className="seconds"/>
+              </div>
+            </article>
+            Waiting for the final step
+            {waitingForConfirmation ? (<progress/>) : (<progress max="120" value={this.props.registeringProgress}/>)}
+          </div>
+        </div>
+      ),
+      readyToRegister: (
+        <div className="domainRegisterProgress domainRegisterAnimation">
+          <svg
+            className="checkmark"
+            xmlns="http://www.w3.org/2000/svg"
+            width="96"
+            height="96"
+            viewBox="0 0 52 52">
+            <circle className="checkmark__circle" cx="26" cy="26" r="25" fill="none" />
+            <path className="checkmark__check" fill="none" d="M14.1 27.2l7.1 7.2 16.7-16.8" />
+          </svg>
+          <div>
+            We are ready!!!
```

Comment:
Lets put just one exclamation mark :P

Reply:
All this together with the other feedback issues are related to the designs, i just made this pretty similar to the designs, actually i did it as most similar as possible, so maybe that's the issue, i think they are going to put this through a designer so after that these kind of issues are going to disapear